docs/src/__examples__/Textarea/DEFAULT.tsx

```diff
@@ -8,9 +8,68 @@ export default {
       placeholder="What I liked about booking with Kiwi.com was ..."
     />
   ),
-  info: {
-    title: "Default textarea",
-    description:
-      "By default, the text area will be a normal-sized space for text that can be resized vertically.",
-  },
+  exampleKnobs: [
+    {
+      component: "Textarea",
+      knobs: [
+        {
+          name: "disabled",
+          type: "boolean",
+          defaultValue: false,
+        },
+        {
+          name: "required",
+          type: "boolean",
+          defaultValue: false,
+        },
```

Comment:
I checked inside Textarea, it appears that, even though it takes `required`, it doesn't do anything with it, oops! 😅

Reply:
yep, should be fixed there :D